src/NodeVisitor/RootNamespaceNormalizer.php

```diff
@@ -23,7 +23,15 @@
 class RootNamespaceNormalizer extends NodeVisitorAbstract
 {
     /**
-     * {@inheritdoc}
+     * Called once before traversal.
+     *
+     * Return value semantics:
+     *  * null:      $nodes stays as-is
+     *  * otherwise: $nodes is set to the return value
+     *
+     * @param Node[] $nodes Array of nodes
+     *
+     * @return null|Node[] Array of nodes
```

Comment:
This is really a failed attempt to fix a false-positive in Scrutinizer. I merely copied the original function docblock from PhpParser. The issue is that Scrutinizer is letting the less specific type hint `array` take precedence over the compatible and more specific `@param Node[]`, and saying that `array` does not satisfy the specification `@return null|Node[]`. I expect this change should actually be reverted, but I didn't want to do so without feedback.

As far as the false-positive, I'm trying to notify Scrutinizer about it, but haven't gotten any confirmation yet.